lib/install/config/shared.js

```diff
@@ -16,15 +17,25 @@ module.exports = {
 
   module: {
     loaders: [
-      { test: /\.coffee$/, loader: "coffee-loader" },
+      { test: /\.coffee(.erb)?$/, loader: "coffee-loader" },
       {
-        test: /\.js$/,
+        test: /\.js(.erb)?$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
         query: {
           presets: ['es2015']
         }
       }
+    ],
+
+    preLoaders: [
+      {
+        test: /\.erb$/,
+        loader: 'rails-erb-loader',
+        query: {
+          rails: '../bin/rails'
```

Comment:
After of getting https://github.com/usabilityhub/rails-erb-loader/pull/21 this needs to be changed to `runner: '../bin/rails runner'`